src/aiai_eval/task.py

```diff
@@ -1 +1,55 @@
 """Abstract Task class."""
+
+from abc import ABC
+from typing import Dict
+
+from datasets import load_metric
+
+from .config import DatasetTask, EvaluationConfig
+
+
+class EvaluationDataset(ABC):
+    """Abstract evaluation dataset class.
+    Args:
+        dataset_task (DatasetTask):
+            The configuration of the dataset task.
+        evaluation_config (EvaluationConfig):
+            The configuration of the evaluation.
+    Attributes:
+        dataset_task (DatasetTask):
+            The configuration of the dataset task.
+        evaluation_config (EvaluationConfig):
+            The configuration of the evaluation.
+    """
+
+    def __init__(self, dataset_task: DatasetTask, evaluation_config: EvaluationConfig):
```

Comment:
Weren't these two configs merged?

Reply:
ScandEval `DatasetConfig` and `DatasetTask` where merged, but not these.